lib/constable_web/templates/layout/app.html.eex

```diff
@@ -19,33 +19,27 @@
       <header class="main-header">
         <%= link "#Constable", to: "/", class: "logo" %>
 
-        <ul class="navigation right">
-          <li>
-            <%= form_tag search_path(@conn, :show), method: :get do %>
-              <input
-                class="header-search-input"
-                name="query"
-                placeholder="Search announcements..."
-                type="text"
-                value="<%= @conn.params["query"] %>"
-              />
-            <% end %>
-          </li>
-          <li>
-            <%= link to: announcement_path(@conn, :new), class: "button button-small button-secondary button-icon" do %>
-              <img src="/images/icon-plus-white.svg" />
-            <% end %>
-          </li>
-          <li>
-            <%= link to: settings_path(@conn, :show), class: "settings-gravatar remote-append" do %>
-              <img
-                src="<%= gravatar(@conn.assigns.current_user) %>"
-                class="current-user-gravatar"
-              />
-            <% end %>
-          </li>
-        </ul>
+        <%= form_tag search_path(@conn, :show), class: "header-search-form", method: :get do %>
+          <input
+            class="header-search-input"
+            name="query"
+            placeholder="Search Constable..."
+            type="text"
```

Comment:
Not sure if this'll mess with the functionality at all, but the type on this should be `search`

Reply:
haha that totally busted things 🙉

Reply:
oh n/m it's just a selector issue sheewwww

Reply:
lolol

re: `sheewwww` i just made that noise in my head as i read it